test/compilable/imports/ddoc_any_postscript.d

```diff
@@ -0,0 +1,10 @@
+module imports.ddoc_any_postscript;
```

Comment:
I strongly recommend only converting one test per PR.  I could be wrong, but I suspect if this PR gets too complicated, it's going to become too controversial and kill the whole idea.

Reply:
yeah, I'll leave it at just the 2 for now.